package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-example-ssl",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Just tried again, looks like it works fine with the versions of `loopback` and `loopback-boot` specified in the `package.json`. I guess `2.1.0` should do? Since we are introducing a new file to set up a part of the app.